src/Microsoft.Health.Fhir.R4.Core/Features/Definition/unsupported-search-parameters.json

```diff
@@ -0,0 +1,131 @@
+        "http://hl7.org/fhir/SearchParameter/OperationDefinition-base",
+        "http://hl7.org/fhir/SearchParameter/OperationDefinition-input-profile",
+        "http://hl7.org/fhir/SearchParameter/OperationDefinition-output-profile",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-composed-of",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-definition",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-depends-on",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-derived-from",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-predecessor",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-successor",
+        "http://hl7.org/fhir/SearchParameter/PlanDefinition-version",
+        "http://hl7.org/fhir/SearchParameter/Procedure-instantiates-canonical",
+        "http://hl7.org/fhir/SearchParameter/Questionnaire-version",
+        "http://hl7.org/fhir/SearchParameter/QuestionnaireResponse-questionnaire",
+        "http://hl7.org/fhir/SearchParameter/RequestGroup-instantiates-canonical",
+        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-composed-of",
+        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-depends-on",
+        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-derived-from",
+        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-predecessor",
+        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-successor",
+        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-version",
+        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-composed-of",
+        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-depends-on",
+        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-derived-from",
+        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-predecessor",
+        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-successor",
+        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-version",
+        "http://hl7.org/fhir/SearchParameter/RiskEvidenceSynthesis-version",
+        "http://hl7.org/fhir/SearchParameter/SearchParameter-component",
+        "http://hl7.org/fhir/SearchParameter/SearchParameter-derived-from",
+        "http://hl7.org/fhir/SearchParameter/ServiceRequest-instantiates-canonical",
+        "http://hl7.org/fhir/SearchParameter/StructureDefinition-base",
+        "http://hl7.org/fhir/SearchParameter/StructureDefinition-base-path",
+        "http://hl7.org/fhir/SearchParameter/StructureDefinition-ext-context",
+        "http://hl7.org/fhir/SearchParameter/StructureDefinition-path",
+        "http://hl7.org/fhir/SearchParameter/StructureDefinition-valueset",
+        "http://hl7.org/fhir/SearchParameter/Subscription-url",
+        "http://hl7.org/fhir/SearchParameter/SubstanceSpecification-code",
+        "http://hl7.org/fhir/SearchParameter/TestScript-version"
+    ],
+    "PartialSupport": [
```

Comment:
Should this be in a different file, or maybe change the name of the file, since the name implies that only unsupported parameters are in this file?

Reply:
I see your point, but is it PartiallySupported or PartiallyUnsupported?

Reply:
Reading though the rest of the PR I agree this is the best place for them to keep all the references in one place. The file name still bugs me, but I can't think of anything better.